console/src/main/scripts/plugins/directives/vendor/moment/moment.d.ts

```diff
@@ -1,472 +0,0 @@
-// Type definitions for Moment.js 2.8.0
```

Comment:
The problem is in the hawt.io dependencies. The hawt.io packages contain their own definitions, which cause conflicts.